quickstart/201-azfw-with-ipgroups/main.tf

```diff
@@ -0,0 +1,273 @@
+    destination_port_range     = "22"
+    source_address_prefix      = "*"
+    destination_address_prefix = "*"
+  }
+}
+
+resource "azurerm_network_interface_security_group_association" "vm_server_nsg_association" {
+  network_interface_id      = azurerm_network_interface.vm_server_nic.id
+  network_security_group_id = azurerm_network_security_group.vm_server_nsg.id
+}
+
+resource "azurerm_network_interface_security_group_association" "vm_jump_nsg_association" {
+  network_interface_id      = azurerm_network_interface.vm_jump_nic.id
+  network_security_group_id = azurerm_network_security_group.vm_jump_nsg.id
+}
+
+resource "azurerm_linux_virtual_machine" "vm_server" {
+  name                            = "server-vm"
+  resource_group_name             = azurerm_resource_group.rg.name
+  location                        = azurerm_resource_group.rg.location
+  size                            = var.virtual_machine_size
+  admin_username                  = var.admin_username
+  admin_password                  = random_password.password.result
+  disable_password_authentication = false
+  network_interface_ids           = [azurerm_network_interface.vm_server_nic.id]
+  os_disk {
+    caching              = "ReadWrite"
+    storage_account_type = "Standard_LRS"
+  }
+  source_image_reference {
+    publisher = "Canonical"
+    offer     = "UbuntuServer"
+    sku       = "18.04-LTS"
+    version   = "latest"
+  }
+  boot_diagnostics {
+    storage_account_uri = azurerm_storage_account.sa.primary_blob_endpoint
+  }
+}
+
+resource "azurerm_linux_virtual_machine" "vm_jump" {
+  name                            = "jump-vm"
+  resource_group_name             = azurerm_resource_group.rg.name
+  location                        = azurerm_resource_group.rg.location
+  size                            = var.virtual_machine_size
+  admin_username                  = var.admin_username
+  admin_password                  = random_password.password.result
+  disable_password_authentication = false
+  network_interface_ids           = [azurerm_network_interface.vm_jump_nic.id]
+  os_disk {
+    caching              = "ReadWrite"
+    storage_account_type = "Standard_LRS"
+  }
+  source_image_reference {
+    publisher = "Canonical"
+    offer     = "UbuntuServer"
+    sku       = "18.04-LTS"
+    version   = "latest"
+  }
+  boot_diagnostics {
+    storage_account_uri = azurerm_storage_account.sa.primary_blob_endpoint
+  }
+  computer_name = "JumpBox"
+
+}
+
+resource "azurerm_route_table" "rt" {
+  name                          = "rt-azfw-eus"
+  location                      = azurerm_resource_group.rg.location
+  resource_group_name           = azurerm_resource_group.rg.name
+  disable_bgp_route_propagation = false
+  route {
+    name                   = "azfwDefaultRoute"
+    address_prefix         = "0.0.0.0/0"
+    next_hop_type          = "VirtualAppliance"
+    next_hop_in_ip_address = azurerm_firewall.fw.ip_configuration.0.private_ip_address
```

Comment:
Could we use `azurerm_firewall.fw.ip_configuration[0].private_ip_address` instead?

Reply:
i can make the change.

Reply:
change made